### PR TITLE
ready home page

### PR DESCRIPTION
**Pull request by artempenteskul**

### CODE REVIEW

**Review by AndriiSkavreniuk — APPROVED** [review]

File: library/catalog/templates/books.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1>Library page</h1>
                 <div class="zoom">
                     <a href={% url 'book-details' book.id %}>
                         <img style="padding: 10px 10px 10px 10px;max-height: 270px; width: 90%" src={{ book.link_img }}>
-                        <p style="margin-left: 10px">{{ book.title }}</p>
+                        <p style="margin-left: 10px; color: #323232">{{ book.title }}</p>
```
Comment (ivan-dubei) [inline library/catalog/templates/books.html:15]: Can we move CSS to the main CSS file ?

File: library/catalog/templates/home.html [hunk 2]
```diff
@@ -6,27 +6,29 @@
...
       <div class="swiper-wrapper">
         <!-- Slides -->
           {% for book in top_books %}
             <div class="swiper-slide">
-                <a href={% url 'book-details' book.id %}>
+                <a href="{% url 'book-details' book.id %}">
                     <img src="{{ book.link_img }}" alt="{{ book.title }}">
-                    <p><b>{{ book.title }}</b></p>
-                    <p>{{ book.author_id.name }}</p>
+                    <p style="color: #323232"><b>{{ book.title }}</b></p>
```
Comment (ivan-dubei) [inline library/catalog/templates/home.html]: Can we move CSS to the main CSS file ?

File: library/catalog/templates/home.html [hunk 3]
```diff
@@ -37,23 +39,26 @@ <h2 class="h2-home-page">New books on a website</h2>
           <div class="swiper-slide">
               <a href={% url 'book-details' book.id %}>
                   <img src="{{ book.link_img }}" alt="{{ book.title }}">
-                  <p><b>{{ book.title }}</b></p>
-                  <p>{{ book.author_id.name }}</p>
+                  <p style="color: #323232; text-decoration: none"><b>{{ book.title }}</b></p>
```
Comment (ivan-dubei) [inline library/catalog/templates/home.html]: Can we move CSS to the main CSS file ?

File: library/catalog/templates/home.html [hunk 4]
```diff
@@ -37,23 +39,26 @@ <h2 class="h2-home-page">New books on a website</h2>
...
-<!--    Search by category section-->
+<div class="container">
+    <!--    Search by category section-->
     <h2 class="h2-home-page">Search your favourite book</h2>
-    <div class="search">
-        <input type="text" class="form-control" placeholder="Title/ Author/ Year" style="width: 85%">
-        <input type="submit" class="btn btn-info" value="Search" style="width: 10%">
-    </div>
+        <form class="search" action="{% url 'form-books' %}">
+            <input type="text" class="form-control" placeholder="Title/ Author/ Year" style="width: 85%" name="searchbar">
```
Comment (ivan-dubei) [inline library/catalog/templates/home.html]: Can we move CSS to the main CSS file ?

File: library/catalog/templates/home.html (hunk 2 above)
Comment (artempenteskul) [reply library/catalog/templates/home.html]: done

File: library/catalog/templates/home.html (hunk 3 above)
Comment (artempenteskul) [reply library/catalog/templates/home.html]: done

File: library/catalog/templates/home.html (hunk 4 above)
Comment (artempenteskul) [reply library/catalog/templates/home.html]: done

File: library/catalog/templates/books.html (hunk 1 above)
Comment (artempenteskul) [reply library/catalog/templates/books.html:15]: I will change the books.html template in the next pull request, there is a lot of CSS on it

**Review by ivan-dubei — APPROVED** [review]

**Review by mehalyna — APPROVED** [review]

**Pull request merged**